app-text/alanvar/additional-files/alanvar.rdef.in

```diff
@@ -0,0 +1,41 @@
+resource app_flags B_MULTIPLE_LAUNCH;
+
+resource app_version {
+	major  = @MAJOR@,
+	middle = @MIDDLE@,
+	minor  = 1,
```

Comment:
> local MINOR is missing

@diversys: [Good catch](https://github.com/haikuports/haikuports/pull/3041#discussion_r217752495), thanks!

So, obviously, the fix is:
```diff
 resource app_version {
 	major  = @MAJOR@,
 	middle = @MIDDLE@,
-	minor  = 1,
+	minor  = @MINOR@,
```